### PR TITLE
More than one window to record

### PR DESCRIPTION
**Pull request by rachelcarmena**
Users can indicate the number of windows to record. While new windows are selected, values of coordinates and dimensions are recalculated.

### CODE REVIEW

**Comment by rhcarvalho** [comment]
Hi @rachelcarmena, thanks for the PR.
While this may be useful to some people, I imagine it is not the most common case (and least it was not part of my original use case).

How about we make it opt-in via an env var or flag? That way we avoid having to fill in one more pop up.

Alternatively, it seems that the multi-window is used just so that we can have a bigger area recorded, did you consider some kind of drag-and-drop to mark which area of the screen to record?

**Comment by rachelcarmena** [comment]
Hi @rhcarvalho, thanks, it's a good idea to activate it via env var or flag.
I didn't consider drag-and-drop, because xwininfo provided me more precision with area selection and coordinates.

File: byzanz-gui [hunk 1]
```diff
@@ -21,28 +21,73 @@ FOLDER="$HOME/Pictures"
...
+        W=$tmpW
+        H=$tmpH
+        X=$tmpX
+        Y=$tmpY
+    fi
+done
+
+# Recovering higher window bar
+Y=$(($Y-25))
+H=$(($H+25))
```
Comment (rhcarvalho) [inline byzanz-gui:90]: Hmm, this is too system-specific, is there a way this could be computed?

**Comment by rhcarvalho** [comment]
I'm closing this in favor of #6 that has been merged. It added a custom copy of the script with support for region selection, which might be a simpler way to solve the use case of recording multiple windows.

I've also added a [list of alternative software for recording GIFs that promise more features](https://github.com/rhcarvalho/byzanz-gui/blob/b08ea8ef78b63892a4ecfe5da524b2d13e58d9ee/README.md#see-also).

In particular, [Peek](https://github.com/phw/peek) looks really interesting.

**Pull request closed without merging**